proto/messages.proto

```diff
@@ -61,7 +61,7 @@ message DeploymentGroup {
   repeated DeploymentGroupSecret secrets = 4;
   string clustering = 5;
   string zta_provider = 6;
-  string zta_key = 7;
+  reserved "zta_key";
```

Comment:
Because the server may still receive this from old clients, I would keep it as it was with a TODO to remove (at the same time we remove the whole entries from zta_provider).